src/lib/SVGMap/SVGMap.stories.svelte

```diff
@@ -0,0 +1,153 @@
+
+  let airQualityScale = scaleQuantile()
+    .domain(county_air_quality.features.map((d) => d.properties.index_air_quality))
+    .range(urbanColors.getDivergingColors());
+
+  const us_cities_geo = topojson.feature(us_cities, "us_cities");
+
+  // cleveland biking
+  const cleveland_bike_data = topojson.feature(cleveland_bike_data_topo, "cleveland_bike_to_work");
+  const clevelandBikeScale = scaleQuantize()
+    .domain(extent(cleveland_bike_data.features, (d) => d.properties.bike_to_work))
+    .range(urbanColors.getMapBlues());
+
+  let clevelandHighlight = "39035197700";
+
+  // pa population
+  const pa_population_geo = topojson.feature(pa_population_topo, "pa_county_population");
+  const paPopRadiusScale = scaleSqrt()
+    .domain(extent(pa_population_geo.features, (d) => d.properties.value))
+    .range([2, 50]);
+</script>
+
+<Template let:args>
+  <SVGMap {...args}>
+    <PolygonLayer />
+    <PointLayer />
+    <LabelLayer getLabel={(d) => d.properties.STUSPS} />
+  </SVGMap>
+</Template>
+
+<Story
+  name="Simple map"
+  source={`<SVGMap projection={geoAlbersUsa} features={states.features}>
+  <PolygonLayer />
+  <PointLayer />
+  <LabelLayer getLabel={(d) => d.properties.STUSPS}/>
+</SVGMap>`}
+>
+  <SVGMap projection={geoAlbersUsa} features={states.features}>
+    <PolygonLayer />
+    <PointLayer />
+    <LabelLayer getLabel={(d) => d.properties.STUSPS} />
+  </SVGMap>
+</Story>
+
+<Story name="Custom colors and multiple layers">
+  <SVGMap projection={geoAlbersUsa} features={county_air_quality.features}>
+    <PolygonLayer
+      fill={(d) => airQualityScale(d.properties.index_air_quality)}
+      stroke={urbanColors.gray_shade_dark}
+      hoverStroke={urbanColors.magenta}
+      hoverStrokeWidth={2}
+    />
+    <PointLayer features={us_cities_geo.features} fill={urbanColors.gray_shade_lighter} />
+    <LabelLayer
+      features={us_cities_geo.features}
+      getLabel={(d) => d.properties.name}
+      fontSize={13}
+    />
+  </SVGMap>
+</Story>
+
+<Story name="Zoomable">
+  <SVGMap scrollWheel="ctrl" zoomable projection={geoMercator} features={nyc_income.features}>
+    <PolygonLayer
+      fill={(d) => incomeScale(d.properties.estimate)}
+      stroke={(d) => incomeScale(d.properties.estimate)}
+      hoverStroke={urbanColors.gray_shade_darker}
+      hoverStrokeWidth={2}
+    />
+    <LabelLayer
+      features={nyc_neighborhoods.features}
+      getLabel={(d) => d.properties.ntaname}
+      minZoom={3}
+    />
+  </SVGMap>
+</Story>
+
+<Story name="Feature highlight">
+  <SVGMap features={cleveland_bike_data.features} projection={geoMercator}>
+    <PolygonLayer
+      fill={(d) => clevelandBikeScale(d.properties.bike_to_work)}
+      stroke="white"
+      hoverStroke={urbanColors.yellow}
+      hoverStrokeWidth={2}
+      highlightFeature={(d) => d.properties.GEOID === clevelandHighlight}
+      on:click={(e) => (clevelandHighlight = e.detail.props.GEOID)}
```

Comment:
What do you think about "clicking on active element toggles it off"?
```
on:click={(e) => {
  if (clevelandHighlight === e.detail.props.GEOID) {
    clevelandHighlight = null;
  } else {
    clevelandHighlight = e.detail.props.GEOID;
  }
}}
```

Reply:
added!